Resources/config/form-fields/header.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" ?>
+<properties xmlns="http://schemas.sulu.io/template/template"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd"
+>
+    <property name="required" type="checkbox" colspan="6">
+        <meta>
+            <title>sulu_form.required</title>
+        </meta>
+        <params>
+            <param name="type" value="toggler"/>
+        </params>
+    </property>
+    <property name="width" type="single_select" mandatory="true" colspan="6">
+        <meta>
+            <title>sulu_form.width</title>
+        </meta>
+        <params>
+            <param name="label">
+                <meta>
+                    <title>sulu_form.width</title>
+                </meta>
+            </param>
+            <param name="default_value" value="0"/>
+            <param name="values" type="collection">
+                <param name="0">
```

Comment:
Name should be 'full' instead of '0' so that's compatible with Form/Type/DynamicFormType.php

Reply:
Fixed. Thanks for pointing this out.